subprojects/enterprise/build.gradle.kts

```diff
@@ -7,6 +7,7 @@ dependencies {
 
     implementation(libs.jsr305)
     implementation(libs.inject)
+    implementation(libs.groovyJson)
```

Comment:
This doesn't seem right. There are test fixtures that need this, but the main code shouldn't.

Can you share what happens without this?

Reply:
Indeed, this was only used in the fixtures. Removing the dependency from here, thanks for catchign